Modules/@babylonjs/react-native/EngineView.tsx

```diff
@@ -9,6 +9,7 @@ export interface EngineViewProps extends ViewProps {
     camera?: Camera;
     displayFrameRate?: boolean;
     isTransparent?: boolean;
+    MSAA: number;
```

Comment:
Narrow this from any number to specific valid numbers, and also make it optional:
```suggestion
    antiAliasing?: 0 | 1 | 2 | 4 | 8 | 16;
```